examples/skinned-mesh.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>THREE.IK - skinned mesh test</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <style>
+        body {
+            font-family: Monospace;
+            background-color: #000;
+            color: #fff;
+            margin: 0px;
+            overflow: hidden;
+        }
+
+        #info {
+            color: #fff;
+            position: absolute;
+            top: 10px;
+            width: 100%;
+            text-align: center;
+            z-index: 100;
+            display: block;
+        }
+
+        #info a {
+            color: #046;
+            font-weight: bold;
+        }
+
+    </style>
+</head>
+
+<body>
+    <script src="scripts/dat.gui.js"></script>
+    <script src="scripts/three.js"></script>
+    <script src="scripts/OrbitControls.js"></script>
+    <script src="scripts/FBXLoader.js"></script>
+    <script src="scripts/IKApp.js"></script>
+    <script src="../build/three-ik.js"></script>
+    <script>
+        class SingleTargetApp extends IKApp {
+            setupGUI() {
+                this.config.constraintType = 'ball';
+                this.config.constraintAngle = 90;
+
+            }
+
+            setupIK() {
+
+                var fbxLoader = new THREE.FBXLoader()
+                fbxLoader.load('assets/rigTest-03.fbx', (rigGroup) => {
+                this.scene.add(rigGroup)
+
+                var rigMesh = rigGroup.children[0];
+                rigMesh.material = new THREE.MeshPhysicalMaterial({
+                  skinning: true,
+                  wireframe: true
+                })
+
+                this.target = new THREE.Object3D()
+                this.target.position.set(1,0,5)
+                this.scene.add(this.target)
+
+                var boneGroup = rigGroup.children[1].children[0];
+                var ik = new IK.IK();
+                const chain = new IK.IKChain();
+                var currentBone = boneGroup
+                for (let i = 0; i < 4; i++) {
+                  currentBone = currentBone.children[0]
```

Comment:
`currentBone` doesn't appear to change over the loop, setting multiple joints to the same bone

Reply:
hm, if you print out the joint array at the end, each joint as a bone with a unique UUID. I've noticed this kind of bug with console.logging before, I think it's a real issue :0 

https://code-maven.com/logging-javascript-objects